### DISCUSSION TITLE
[Feature request] Show repository size on the repository homepage, like GitLab does

### DISCUSSION THREAD

**Discussion by seanpm2001** [community/community#124014] [Repositories]
### Select Topic Area

Product Feedback

### Body

***

# [Feature request] Show repository size on the repository homepage, like GitLab does

There is a helpful GitLab feature that GitHub is missing, which is a piece of information on each repository that shows not only the owner, but any collaborator or viewer,  how much storage space the repository is using.

I know you can go to a page on the user profile to see the repository size, but this isn't as helpful (especially when you aren't the owner) and also because that page only lets you view 1000 repositories. I also feel like not enough information is displayed.

***

**Comment by grandmas-favorite** [comment]
Hey @seanpm2001! We appreciate your feedback and your valuable input. We want to be transparent about our current priorities. You can gain insights into the projects we’re currently focusing on for the upcoming year by checking our [public roadmap](https://github.com/github/roadmap/projects/1). 

While smaller fixes may not always be explicitly listed, they often get addressed when related work is in progress. We’d like to emphasize that as a valued GitHub user, your feedback in this Discussion space, along with appropriate tags and examples, plays a crucial role in shaping the development of GitHub. While we can’t guarantee immediate action on every piece of feedback, please rest assured that we are attentive to your suggestions and continuously strive to enhance the GitHub experience.
